src/Symfony/Contracts/Translation/TranslatorTrait.php

```diff
@@ -115,6 +115,10 @@ public function trans($id, array $parameters = [], $domain = null, $locale = nul
                 return strtr($standardRules[0], $parameters);
             }
 
+            if (true === $behaveSilently) {
+                return;
```

Comment:
@Simperfit what about logging before returning (error/critical or other level)?

so that in env dev at least the developer can have a clue

totaly what I imagined, thank you for providing such feature

Reply:
Good catch, now when kernel.debug is true, it will throw, when kernel.debug is false, it will not throw :). So in env dev it will throw and in prod env it will not.
np